### PR TITLE
parseJson fix

### PR DESCRIPTION
**Pull request by aroralanuk**
`vm.parseJson()` needs "." before the first key argument.

### CODE REVIEW

**Comment by aroralanuk** [comment]
@asaj

File: lib/ConfigLib.sol [hunk 1]
```diff
@@ -66,11 +79,11 @@ library ConfigLib {
     ) internal view returns (HyperlaneDomainConfig memory) {
         string memory json = vm.readFile("config/networks.json");
         uint32 domainId = abi.decode(
-            vm.parseJson(json, string.concat(chainName, ".id")),
+            vm.parseJson(json, string.concat(".", chainName, ".id")),
             (uint32)
         );
         address owner = abi.decode(
-            vm.parseJson(json, string.concat(chainName, ".owner")),
+            vm.parseJson(json, string.concat(".", chainName, ".owner")),
```
Comment (0xsuryansh) [inline lib/ConfigLib.sol:86]: @aroralanuk @tkporter
Foundry supports Type Coercion 
Ref : https://book.getfoundry.sh/cheatcodes/parse-json#type-coercion

So we can make use of it like this:

```bash
chainName = string.concat('.',chainName); 
address owner = vm.parseJsonAddress(json, string.concat(chainName,".owner"));
```

Similarly at other places

Comment (chainchampion) [reply lib/ConfigLib.sol:86]: @tkporter same suggestion for https://github.com/hyperlane-xyz/hyperlane-deploy/pull/29

Comment (asaj) [reply lib/ConfigLib.sol:86]: Good suggestion! Unfortunately we're considering moving away from using foundry in this repo, and instead using the ethers-based `HyperlaneCoreDeployer` from the SDK.

**Comment by asaj** [comment]
We've moved on from foundry

**Pull request closed without merging**